x-pack/plugins/infra/public/alerting/inventory/components/alert_flyout.tsx

```diff
@@ -23,10 +23,10 @@ interface Props {
 
 export const AlertFlyout = ({ options, nodeType, filter, visible, setVisible }: Props) => {
   const { triggersActionsUI } = useContext(TriggerActionsContext);
-
-  const { inventoryPrefill } = useAlertPrefillContext();
-  const { customMetrics = [] } = inventoryPrefill ?? {};
   const onCloseFlyout = useCallback(() => setVisible(false), [setVisible]);
+  const { inventoryPrefill } = useAlertPrefillContext();
+  const { customMetrics = [], accountId, region } = inventoryPrefill;
```

Comment:
It seems like some places use `PrefilledInventoryAlertFlyout` and other places `AlertFlyout` directly. It's weird because `PrefilledInventoryAlertFlyout` calls `AlertFlyout`

Reply:
You mean why do we use `useAlertPrefillContext` inside `alert_flyout`?